boards/bytesatwork/bytesensi_l/board.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2024 bytesatwork AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/init.h>
+
+static int enable_1_wire(void)
+{
+	/* Enable 1-wire to fix i2c bus */
+	const struct gpio_dt_spec enable_1_wire_gpio =
+		GPIO_DT_SPEC_GET_OR(DT_PATH(zephyr_user), enable_1wire_gpios, {0});
+
+	if (!gpio_is_ready_dt(&enable_1_wire_gpio)) {
+		return -ENODEV;
+	}
+
+	return gpio_pin_configure_dt(&enable_1_wire_gpio, GPIO_OUTPUT_ACTIVE);
```

Comment:
Why not use `gpio-hog` for this? See for example 12cca8939e3d4eb7143d0a6086dcf600694bbe8d.

Reply:
Awesome, I learned a new trick today, thanks!